Cask

```diff
@@ -7,5 +7,4 @@
   (depends-on "ecukes")
   (depends-on "ert-expectations")
   (depends-on "ert-runner")
-  (depends-on "rx")
-  )
+  (depends-on "rx"))
```

Comment:
`rx` is a built-in package, and shouldn't be declared as a dependency like this.